data/universities.yml

```
@@ -2501,8 +2501,8 @@
 - title: Manav Rachna International Institute of Research and Studies - MRIIRS
   location: Faridabad (India)
   courses:
-  - name: Mobile Computing with Android (CSH319) T & P
-    url: https://drive.google.com/file/d/1xL9raHbwl7WOZ34vxAA-ux4aZGso6nUy/view?usp=sharing
+  - name: BCS-DS-577: Kotlin
```

The colon should be escaped in YML files
```suggestion
  - name: "BCS-DS-577: Kotlin"
```